package.yaml

```diff
@@ -47,7 +53,14 @@ ghc-options:
   - -fno-sort-valid-hole-fits
 
 cpp-options:
-  - -std=c++17
+ - -std=c++17
+
+when:
```

Comment:
I wasn't sure how to do appropiate conditioning in package.yaml. Still I have to look into this.